lib/mp-readline/readline.c

```diff
@@ -98,6 +102,38 @@ typedef struct _readline_t {
 
 STATIC readline_t rl;
 
+#if MICROPY_REPL_EMACS_KEYS
+#define FORWARD true
+#define BACKWARD false
+
+size_t get_word_cursor_pos(bool direction) {
```

Comment:
This can be made a `STATIC` function, since it's local to this file.

Also `direction` can be an `int` taking value 1 or -1, then there's no need to convert it to `cursor_pos_step`, nor have `FORWARD` and `BACKWARD` constants.